doc/releases/reef.rst

```diff
@@ -0,0 +1,219 @@
+====
+Reef
+====
+
+Reef is the 18th stable release of Ceph. It is named after the reef squid
+(sepioteuthis).
+
+v18.2.0 Reef
+============
+
+This is the first stable release of Ceph Reef.
+
+CephFS
+~~~~~~
+
+* CephFS: The ``mds_max_retries_on_remount_failure`` option has been renamed to
+  ``client_max_retries_on_remount_failure`` and moved from ``mds.yaml.in`` to
+  ``mds-client.yaml.in``. This change was made because the option has always
+  been used only by the MDS client.
+* CephFS: It is now possible to delete the recovered files in the
+  ``lost+found`` directory after a CephFS post has been recovered in accordance
+  with disaster recovery procedures.
+
+MGR
+~~~
+
+* mgr/snap_schedule: The snap-schedule mgr module now retains one snapshot less
+  than the number mentioned against the config tunable
+  ``mds_max_snaps_per_dir``. This means that a new snapshot can be created and
+  retained during the next schedule run.
+* The ``ceph mgr dump`` command now outputs ``last_failure_osd_epoch`` and
+  ``active_clients`` fields at the top level. Previously, these fields were
+  output under the ``always_on_modules`` field.
+  
+RADOS
+~~~~~
+
+* The trimming of PGLog dups is now controlled by the size instead of the
+  version. This fixes the PGLog inflation issue that occurred when the on-line
+  (in OSD) trimming got jammed after a PG-split operation. A new off-line
```

Comment:
I favor `online` and `offline`

Reply:
I agree. Thanks for this catch.
Accepted.